### ISSUE TITLE
Menu icon & text overlap for menu with checkbox in dark mode

### ISSUE THREAD

**Issue by zmni** [pbek/QOwnNotes#1221]
OS: `Windows 8.1 (6.3)`. Version: `19.6.4`. Build: `4332`

#### Actual behaviour
Overlaping icon & text.

![a](https://user-images.githubusercontent.com/841523/59964202-d2897e00-9527-11e9-8825-56085dc88164.jpg)
![b](https://user-images.githubusercontent.com/841523/59964204-d7e6c880-9527-11e9-849c-4194df056f14.jpg)
![c](https://user-images.githubusercontent.com/841523/59964205-d7e6c880-9527-11e9-92f9-d6360a61b1dc.jpg)

**Comment by pbek** [comment]
Thank you for reporting, but I can't reproduce that on Windows 10 in German/English. 

Can you please post the output from the debug settings (see issue template).

**Comment by pbek** [comment]
Did you have that problem on previous versions of QOwnNotes?

**Comment by zmni** [comment]
[Debug information is on GitHub Gist](https://gist.github.com/zmni/935a2c494fd66f32a80e0afc97c657d3).

No, I don't have this issue. I don't remember which version I previously installed, but it's old. It works for me, so I always skip the updates.

**Comment by pbek** [comment]
No problem here...

![image](https://user-images.githubusercontent.com/1798101/59967402-96efb380-9529-11e9-8a45-1e7c2689c26f.png)

But I also get the problem in dark mode on Windows 10.

**Comment by zmni** [comment]
Yeah, ditto. Apparently the problem occurred when dark mode was active. I disable dark mode and the problem is gone.

**Comment by pbek** [comment]
## 19.6.5
- fixed icon-text-overlap for checkboxes and radio groups in the main menu in dark mode

**Comment by pbek** [comment]
The icon positioning seems to be handled differently on newer Qt versions...

**Comment by pbek** [comment]
There now is a new release, could you please test it and report if it works for you?

**Comment by zmni** [comment]
Yes, no issue now. No more overlap.

Thank you.

**Comment by pbek** [comment]
Great, thank you for testing!

**Comment by pbek** [comment]
I currently seem to have the problem that I can't see a checked state in dark mode if the menu item has an icon.

![image](https://user-images.githubusercontent.com/1798101/68190467-bf92ba00-ffad-11e9-8a2d-0de678a45c40.png)

**Comment by pbek** [comment]
I'll fix that in the next release.
